v2/tools/generator/internal/astmodel/oneof_type_test.go

```diff
@@ -15,33 +15,33 @@ func TestOneOfEqualityDoesNotCareAboutOrder(t *testing.T) {
 	t.Parallel()
 	g := NewGomegaWithT(t)
 
-	x := NewOneOfType("x", StringType, BoolType)
-	y := NewOneOfType("y", BoolType, StringType)
+	version1 := NewOneOfType("one", StringType, BoolType)
```

Comment:
This is technically a different assert than the one that was here before, because before the name of the type didn't matter and now it does (I see the name didn't matter before in the assert and now it does).

I assume that this is OK because we never have types that differ by names that are supposed to compare as equal (your change seems better than it was before...), but just want to confirm.

Reply:
Correct.